raiden-dapp/tests/unit/components/account/account-content.spec.ts

```diff
@@ -200,4 +200,36 @@ describe('AccountContent.vue', () => {
       }),
     );
   });
+
+  test('displays "Disconnect" button when connected', async () => {
+    const wrapper = await createWrapper(
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
+      undefined,
+    );
+    const disconnectButton = wrapper.find('.account-content__disconnect__button');
+
+    expect(disconnectButton.exists()).toBe(false);
+  });
+
+  test('does not display "Disconnect" button when not connected', async () => {
+    const wrapper = await createWrapper();
```

Comment:
Nitpick: also this is the default, could you explicitly set the connection status parameter to `true`?  So we have literally the condition of the title guaranteed.
I know setting this parameter is annoying. I guess that is my fault because I have written such ugly wrapper functions. The lack of named parameters annoyed. So I started a new concept for unit tests `createWrapper` functions in my recent PR. Though I don't want to ask you to change this here. Just wanted to say sorry. :see_no_evil:

Reply:
Haha, sure, I can add 5 x `undefined` 😂.